htdocs/js/helpHandler.js

```diff
@@ -93,7 +93,7 @@ $(document).ajaxError(function(event, jqxhr, settings, thrownError) {
         };
         $.ajax({
           type: 'post',
-          url: loris.BaseURL + '/main.php',
+          url: loris.BaseURL + '/',
```

Comment:
@driusan I believe the `+ '/'` is unnecessary. Tested your PR without it and couldn't find any issue.

Reply:
Okay, I removed it.. to be honest I'm not sure that any of this part of the code is necessary, I don't know what it's trying to do.

Reply:
@driusan seems to be used for logging in. I think it will be unnecessary when the react login pr is merged.